### PR TITLE
Fix on EntityRendered to support break lines (after last entity) (closes #5838)

### LINKED ISSUE #5838: Displacy's EntityRenderer not breaking line after last entity
## How to reproduce the behaviour

```python
SAMPLE_TEXT = '''First line
Second line, with ent
Third line
Fourth line
'''
nlp = English()
doc = nlp(SAMPLE_TEXT)
doc.ents = [Span(doc, 7, 8, label='test')]
html = displacy.render(doc, style='ent')
```
### Results
Screen Shot 2020-07-29 at 09 31 26

## Your Environment

* **spaCy version:** 2.3.2
* **Platform:** Darwin-19.3.0-x86_64-i386-64bit
* **Python version:** 3.7.4

### PR DESCRIPTION
**Pull request by leyendecker**
…ses #5838

## Checklist

- [x] I have submitted the spaCy Contributor Agreement.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: .github/contributors/leyendecker.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+7. Please place an “x” on one of the applicable statement below. Please do NOT
+mark both statements:
+
+    * [x] I am signing on behalf of myself as an individual and no other person
+    or entity, including my employer, has or will have rights with respect to my
+    contributions.
+
+    * [x] I am signing on behalf of my employer or a legal entity and I have the
+    actual authority to contractually bind that entity.
+
```
Comment (svlandeg) [inline .github/contributors/leyendecker.md:96]: Could you pick just one of the two options?

File: spacy/tests/regression/test_issue5838.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    # not working after last entity
+
+    nlp = English()
+    doc = nlp(SAMPLE_TEXT)
+    doc.ents = [Span(doc, 7, 8, label='test')]
+
+    html = displacy.render(doc, style='ent')
+    found = html.count('</br>')
+    print(html)
+    assert found == 4
```
Comment (svlandeg) [inline spacy/tests/regression/test_issue5838.py:23]: Nice test! Let's just remove the `print` statement :-)

**Comment by leyendecker** [comment]
Thanks for the review @svlandeg. Just pushed another commit.

**Comment by ines** [comment]
Yes, looks good to me, thanks for the fix 👍

**Pull request merged**